modules/ROOT/pages/websockets/websocket-connection-management-components.adoc

```diff
@@ -0,0 +1,65 @@
+= About Websocket Connection Management Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== About Connection Management Components
+
+These components provide tools for connection management for both server and client side operations. All of these operations are available for both Inbound and Outbound sockets.
+
+=== On Socket Closed
```

Comment:
This is way too vague. This needs to explain what is it that you get in the payload and what in the attributes. Example has to be better